### PR TITLE
Form structure

### PR DESCRIPTION
**Pull request by sneeu-leeu**
#### In this project we learned:
* To create a page to display and input data
* Use semantic HTML to create forms
* Use Inputs to collect data
* Use buttons to submit and relay the data collected
* Use CSS to style these forms and data collectives

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        <li>Privacy</li>
+        <li>Security</li>
+    </ul>
+    <p class="footer-paragraph">
+        Legal Privacy Security
+        © 2020 Intuit, Inc. All rights reserved. Intuit, 
+        QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.<br/>
+        Terms and conditions, features, support, pricing, and service options 
+        subject to change without notice.</p>
+</footer>
```
Comment (santiagorodriguezbermudez) [inline Index.html]: - [x] The footer must be inside the body tags. Kindly move it at the end of the body tags.

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    </head>
+
+<body>
+
+    <header>
+        <nav class="navb">
+            <div class="sign-in">
+                <div class="text">
+                <small>Already have an Intuit account?</small> </br>
+                <small style="color: blue">I forgot my User ID or Password</small>
```
Comment (santiagorodriguezbermudez) [inline Index.html]: - [x] Do not use inline styling. Rather, implement this on your CSS file to keep everything organized.

**Review by santiagorodriguezbermudez — CHANGES_REQUESTED** [review]
# Changes Required ♻️

Nice work so far! The page is looking good. 🚀🚀🚀

Please address the following changes:

- [x] Your Pull Request and Commits must be improved. Remember that both must have a descriptive language. In your case you have repeated the name of a couple of commits:

- 'Image' is used twice
- 'changed form footer' is also twice
- 'form-setup' as well. 

Kindly change the name of those commits to make them more descriptive according to the changes they each represent. Check out this tutorial for such think: [How to rebase past commits](https://help.github.com/en/github/using-git/about-git-rebase)

- [x] Your Pull Request must be more descriptive. Describe what are the things that you worked and implemented on the PR. 

- [x] Linters are not working properly. Please setup the linters for this project. Follow the tutorial linked on the project specifications: [Setup Github Actions (aka linters)](https://github.com/microverseinc/linters-config/tree/master/html-css)

- [x] Your top right bottom should be on the corner. Delete the right padding and adjust the color so that it similar to the original project. 
Screen Shot 2020-08-19 at 11 53 29 AM

- [x] Your logos are not centralized horizontally nor vertically align with their corresponding text. Align horizontally and vertically. 
Screen Shot 2020-08-19 at 11 57 51 AM

- [x] Footer should be below the form and not at the button according to the original page.
Screen Shot 2020-08-19 at 12 00 28 PM

- [x] Do not use inline styling change the referred style below on your HTML file. 

- [x] Change the footer to be part of your body tags. 

- [x] The width of the form should be a bit smaller according to the original project.

- [ ] [OPTIONAL]: Nice work deploying on Netlify this will help you out in the future. However, for the purpose of simplicity, I strongly suggest you use [Githack](https://raw.githack.com/) for upcoming projects as it is way simpler and you can focus on learning the core HTML concepts. 

Happy coding 👨🏻‍💻 👩‍💻
If you have additional questions, feel free to reach out on slack @santiagorodriguezb

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    </section>  
+</main>
+  
+
+<footer class="footer-element">
+    <ul class="ul-footer">
+        <li>Legal</li>
+        <li>Privacy</li>
+        <li>Security</li>
+    </ul>
```
Comment (SpaYco) [inline Index.html]: - [x] these should be links

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 

Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

first off, your page looks messed up on my screen unless I change my chrome size, but I understand this is because we have different screen sizes, please let future TSEs who review your projects know through the PR post next time for less confusion, you can also take it a step further and style your page with responsive units such as the [percentage or viewport units](https://www.w3schools.com/cssref/css_units.asp)!

- [x] you have **20** errors from the [HTML Validator](https://validator.w3.org/#validate_by_upload), please fix them

- [x] fix your linting errors please!

- **more changes below**





### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by SpaYco — APPROVED** [review]
## Project Approved ✔️ 

Good Job making the changes required! don't forget to merge and good luck with your next projects! 


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**